clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/BasicAmazonDeployHandler.groovy

```diff
@@ -329,6 +335,35 @@ class BasicAmazonDeployHandler implements DeployHandler<BasicAmazonDeployDescrip
                                                     String sourceAsgName, Boolean useSourceCapacity,
                                                     BasicAmazonDeployDescription description) {
 
+    def copySourceSubnetIdOverrides = description.copySourceSubnetIdOverrides && !description.subnetIds
```

Comment:
worth explicitly checking subnetIds == null vs groovy false? if someone explicitly passed an empty list would that indicate they want the defaults for the subnetType?

Reply:
I'm inclined to leave as is and use `copySourceSubnetIdOverrides` rather than empty subnetIds to get back on the paved road.